arangod/IResearch/ViewSnapshot.h

```diff
@@ -43,17 +43,23 @@ class Methods;
 }  // namespace transaction
 namespace iresearch {
 
+using ViewSegment =
+    std::tuple<DataSourceId, irs::sub_reader const*, StorageSnapshot const&>;
```

Comment:
struct please!

Reply:
Why? )

Reply:
Difficult to understand what's going ok, of course it's not like we cannot merge with tuple, but without tuple it's better.

Just see on your own code like get<0>/etc after few weeks

Do you agree?

Reply:
No. It`s quite simple construct, But if you strongly insist I could make it a struct )